gresources/nemo-file-management-properties.glade

```diff
@@ -1234,6 +1234,23 @@ along with .  If not, see <http://www.gnu.org/licenses/>.
                                         <property name="can-focus">False</property>
                                         <property name="orientation">vertical</property>
                                         <property name="spacing">6</property>
+                                        <child>
+                                          <object class="GtkRadioButton" id="scripts_execute_terminal_radiobutton">
+                                            <property name="label" translatable="yes">_Run executable text files in the terminal when they are opened</property>
+                                            <property name="visible">True</property>
+                                            <property name="can-focus">True</property>
+                                            <property name="receives-default">False</property>
+                                            <property name="use-underline">True</property>
+                                            <property name="xalign">0</property>
+                                            <property name="active">True</property>
+                                            <property name="draw-indicator">True</property>
```

Comment:
You need to have:
```
<property name="group">scripts_execute_radiobutton</property>
```

This is in every entry except `scripts_execute_radiobutton` itself, and ties the other buttons together with that one (otherwise you have 2 buttons active at once).